tskv/src/summary.rs

```diff
@@ -577,15 +584,34 @@ impl Summary {
                     &mut file_metas,
                     min_seq.copied(),
                 );
+
+                let last_seq = new_version.last_seq();
                 let flushed_mem_cahces = mem_caches.get(&tsf_id);
                 tsf.write()
                     .await
                     .new_version(new_version, flushed_mem_cahces);
+
+                if !self.opt.raft_mode {
+                    let (task, rx) = WalTask::new_clear_wal_entry(
+                        tsf.read().await.tenant_database().to_string(),
+                        tsf_id,
+                        last_seq,
+                    );
+
+                    if self.wal_sender.send(task).await.is_ok() {
+                        receivers.push(rx);
+                    }
+                }
+
```

Comment:
we only need to trigger delete wal file on flush job not on all new version generated.